site/tests/universal/page-header.test.ts

```diff
@@ -32,7 +32,7 @@ test("page header navigation works", async ({ page, isMobile }) => {
 
   await expect(
     page.locator(
-      "h1 >> text=Interactive, data-driven blocks to use in your projects",
+      'h1:has-text("Interactive, data-driven blocks to use in your projects")',
```

Comment:
@teenoh the original selector failed in Safari a few times for me, e.g.: https://github.com/blockprotocol/blockprotocol/actions/runs/2711735163

Not sure why because I've seen the element in the trace:
<img width="1722" alt="Screenshot 2022-07-21 at 13 48 29" src="https://user-images.githubusercontent.com/608862/180218210-644f3656-a8e4-4395-b8c5-5d2895fe6faf.png">

Replacing `>> with `:` helped, but maybe I was just lucky.

Reply:
That's weird. I'll take note of this in other tests. Thanks!